python/cugraph/cugraph/structure/graph_classes.py

```diff
@@ -70,7 +70,7 @@ def __init__(self, m_graph=None, directed=False):
                                         edge_attr=weights)
             else:
                 msg = (
-                    "Graph can only be initialized using MultiGraph "
+                    "Graph can only be initialized using cuGraph or networkX MultiGraph object."
```

Comment:
NetworkX MultiGraph graphs are not yet supported.
```
>>> import pandas as pd
>>> import networkx as nx
>>>
>>> import cudf
>>> import cugraph
>>>
>>> df = pd.DataFrame({"src":[0,1,2],"dst":[1,2,0]})
>>>
>>> nxMG = nx.from_pandas_edgelist(df, create_using=nx.MultiGraph,
...                                source="src", target="dst")
>>>
>>> cgMG = cugraph.MultiGraph()
>>> cgMG.from_pandas_edgelist(df, source="src", destination="dst")
>>>
>>> G = cugraph.Graph(m_graph=nxMG)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/conda/envs/rapids/lib/python3.8/site-packages/cugraph-21.12.0a0+57.gcb65876a-py3.8-linux-x86_64.egg/cugraph/structure/graph_classes.py", line 62, in __init__
    elist = m_graph.view_edge_list()
AttributeError: 'MultiGraph' object has no attribute 'view_edge_list'
>>>
>>> G = cugraph.Graph(m_graph=cgMG)
>>>
```
```suggestion
                    "Graph can only be initialized using a cugraph.MultiGraph object."
```